app/src/main/java/io/novafoundation/nova/app/root/data/browser/TabsRepository.kt

```diff
@@ -0,0 +1,16 @@
+package io.novafoundation.nova.app.root.data.browser
+
+import io.novafoundation.nova.common.utils.mapList
+import io.novafoundation.nova.core_db.dao.BrowserTabsDao
+import kotlinx.coroutines.flow.Flow
+
+class TabsRepository(private val tabsDao: BrowserTabsDao) {
```

Comment:
This should have an interface in dapp browser api and implementation in dapp browser impl and be provided to root